### ISSUE TITLE
Docs needed: bundles and subordinates

### ISSUE THREAD

**Issue by wurde** [juju/docs#1836]
```
ERROR cannot deploy bundle: cannot add unit for application "osquery": cannot add unit 1/1 to application "osquery": cannot add unit to application "osquery": application is a subordinate
```

The above is very unhelpful when looking up how to properly define a subordinate charm within the Bundle syntax. Any examples? Can we just get a complete list of Bundle directives for easy reference?

**Comment by wurde** [comment]
The correct way to deploy subordinate services is to NOT assign them to a machine or declare number of units. This would be super helpful for anyone trying to write architecture code via a Bundle.

    num_units: 1
    to:
      - "0"

**Comment by evilnick** [comment]
Thanks for these points. I thin we could actually do with a proper page on writting/editing bundles. The current documentation is very much centred around consuming them, but of course, many people will want or need to do both

**Comment by pmatulis** [comment]
Closed via #3379
